lib/AST/ASTPrinter.cpp

```diff
@@ -3228,21 +3228,23 @@ class TypePrinter : public TypeVisitor<TypePrinter> {
         llvm::raw_svector_ostream UnderlyingOS(UnderlyingStrVec);
         T->getElementType().print(UnderlyingOS);
       }
-      if (UnderlyingStrVec.startswith("Builtin."))
+      std::string builtin_name = BUILTIN_NAME;
+      std::string builtin_type_prefix = builtin_name + ".";
```

Comment:
Since we dynamically construct this a few times, we should maybe just make a new constant for “Builtin.”

Reply:
Good point. Fixed it.